10326.xml

```diff
@@ -0,0 +1,724 @@
+      </Item>
+      <Item ID="305">
+        <Name>CO</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
+        <Description><![CDATA[CO concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="306">
+        <Name>CO2</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>ppm</Units>
+        <Description><![CDATA[CO2 concentration of the environment.]]></Description>
+      </Item>
+      <Item ID="307">
+        <Name>PM10</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>μg/m3</Units>
+        <Description><![CDATA[PM10 of the environment.]]></Description>
+      </Item>
+	  
+	  
+      <Item ID="601">
+        <Name>Speed</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>M/H</Units>
+        <Description><![CDATA[The speed of the robot.]]></Description>
+      </Item>
+	  <Item ID="602">
+        <Name>Water Used</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[The percentage of water used for CloudCleanX.]]></Description>
+      </Item>
+	  <Item ID="603">
+        <Name>Dust Box Used</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration>0..100</RangeEnumeration>
+        <Units>%</Units>
+        <Description><![CDATA[The percentage of Dust Box Space used for CloudCleanX.]]></Description>
+      </Item>
+	  <Item ID="604">
+        <Name>Obstacle Distance</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>cm</Units>
+        <Description><![CDATA[The distance between the robot and the obstacle.]]></Description>
+      </Item>
+	  
+	  <Item ID="607">
+        <Name>Robot Temperate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
Have commented before.